colossalai/kernel/triton/flash_decoding.py

```diff
@@ -124,6 +124,129 @@ def _flash_decoding_fwd_kernel(
     tl.store(mid_o_lse + offsets_mid_o_lse, m + tl.log(l))
 
 
+# Triton 2.1.0
+@triton.jit
+def _alibi_flash_decoding_fwd_kernel(
+    Q,  # [batch_size * q_len, head_num, head_dim]
+    KCache,  # [num_blocks, num_kv_heads, block_size, head_dim]
+    VCache,  # [num_blocks, num_kv_heads, block_size, head_dim]
+    block_tables,  # [batch_size, max_blocks_per_sequence]
+    mid_o,  # [batch_size * q_len, head_num, kv_split_num, head_dim]
+    mid_o_lse,  # [batch_size * q_len, head_num, kv_split_num]
+    kv_seq_len,  # [batch_size]
+    q_len,
```

Comment:
Please add a NOTE or TODO to notify people that alibi flash decoding does not support q_len > 1 at this moment

Reply:
Done